website/templates/project/addons.mako

```diff
@@ -116,6 +116,26 @@
                         % if addon.get('node_settings_template'):
                             ${render_node_settings(addon)}
                         % endif
+                        % if addon['addon_short_name'] == 'github':
+                            <div class="dismissible-alerts hidden" data-bind="css: {'hidden': $root.isDismissed('githubOrgs')}">
+                                <div class="alert alert-info alert-dismissible" role="alert">
+                                    <button type="button" class="close" data-dismiss="alert" aria-label="Close"
+                                        data-bind="tooltip: {title: 'Once dismissed, this alert will not be shown again.', placement: 'left'}, click: $root.dismiss.bind($root, 'githubOrgs')">
```

Comment:
This tooltip is unnecessary--as a user, I would expect clicking the "x" to dismiss the alert, so this just adds noise.